core/lib.rs

```diff
@@ -8,7 +8,7 @@ pub mod arena;
 mod async_cancel;
 mod async_cell;
 pub mod convert;
-pub mod cppgc;
+mod cppgc;
```

Comment:
Why remove `pub`? `make_cppgc_object` and `try_unwrap_cppgc_object` is used in node:vm